src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -362,6 +362,31 @@ public void KeyboardInput_MemoryHotkeys()
             Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("MemoryPaneEmpty")); //verifies the Memory panel's memory clear button hotkey
         }
 
+        [TestMethod]
+        [Priority(1)]
+        public void KeyboardInput_HistoryHotkeys()
+        {
+            //Verifies history buttons
+            CalculatorApp.EnsureCalculatorHasFocus();
+            CalculatorApp.Window.SendKeys("4");
+            CalculatorApp.Window.SendKeys(Keys.Multiply);
+            CalculatorApp.Window.SendKeys("5");
+            CalculatorApp.Window.SendKeys(Keys.Divide);
+            CalculatorApp.Window.SendKeys("6");
+            CalculatorApp.Window.SendKeys(Keys.Equal);
+            page.HistoryPanel.ResizeWindowToDisplayHistoryButton();
+            CalculatorApp.Window.SendKeys(Keys.Control + "h" + Keys.Control);
+            var historyFlyoutItems = page.HistoryPanel.GetAllHistoryFlyoutListViewItems();
+            Assert.IsTrue(historyFlyoutItems[0].GetValue().Equals("3.333333333333333", StringComparison.InvariantCultureIgnoreCase)); //verifies History button hotkeys
+            Assert.IsTrue(historyFlyoutItems[0].GetExpression().Equals("4   ×   5   ÷   6 =", StringComparison.InvariantCultureIgnoreCase)); //verifies History button hotkeys
```

Comment:
The test shouldn't be sensitive to the whitespace in this expression. Let's remove the spaces...

```
var historyExpression = Regex.Replace(historyFlyoutItems[0].GetExpression(), @"\s", string.Empty);
```

...and then test the result:

```
Assert.IsTrue(historyExpression.Equals("4×5÷6=", StringComparison.InvariantCultureIgnoreCase));
```

Reply:
Edit made in commit 4f176b8535caa875783464bc421892a5c1849552